cmd/globals.go

```diff
@@ -393,6 +393,10 @@ var (
 	// Is _MINIO_DISABLE_API_FREEZE_ON_BOOT set?
 	globalDisableFreezeOnBoot bool
 
+	// Find out which NIC the instance is running on based on Endpoint
```

Comment:
```suggestion
	// Contains NIC interface name used for internode communication
```

Are we sure there is only ever 1?